modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +373,152 @@ ID HOST PORT RACK
 
 endif::[]
 
+== Multi-AZ deployments
+
+As mentioned previously, in a multi-AZ deployment a single Redpanda cluster has brokers distributed over multiple availability zones. The basis for this architecture is Redpanda’s rack awareness feature, in which each zone becomes a “rack”. This allows Redpanda to place replicas with respect to failure domains, resulting in a cluster that can survive a zone outage.
+
+NOTE: Adding a zone does not necessarily increase availability - it’s the replication factor of a given partition that matters. If all of your partitions use a replication factor of three, then adding an additional broker in a fourth zone just means fewer partitions are affected by an outage (since the workload is spread out more).
```

Comment:
```suggestion
NOTE: Adding a zone does not necessarily increase availability. The replication factor of a given partition is most important. If all of your partitions use a replication factor of three, then adding an additional broker in a fourth zone just means fewer partitions are affected by an outage (since the workload is more spread out).
```